openshift/cirras-underwriting-api-configmap.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   namespace: '#{NAMESPACE}#'
 immutable: false
 data:
+  TOMCAT_PORT: '#{TARGET_PORT}#'
+  TOMCAT_MAX_THREADS: '#{TOMCAT_MAX_THREADS}#'
```

Comment:
I don't see a TOMCAT_MAX_THREADS or CONNECTION_TIMEOUT in openshift-deploy.yml? Or is that not where they come from?

Reply:
Looks like TOMCAT_MAX_THREADS is fixed, but what about CONNECTION_TIMEOUT?

Reply:
Added missing property to openshift-deploy.yml